lib/PaymentService.js

```diff
@@ -0,0 +1,79 @@
+/**!
+ * Copyright (c) 2018-2019 Digital Bazaar, Inc. All rights reserved.
+ */
+'use strict';
+
+import axios from 'axios';
+
+/**
+ * Wrapper around the bedrock-payment-http API.
+ *
+ * @param {object} [options] - Options to use.
+ * @param {string} [options.baseURL = '/payment'] - The baseURL for
+ *   the payment api.
+ *
+ * @returns {PaymentService} The PaymentService.
+ */
+export default class PaymentService {
+  constructor({baseURL = '/payment'} = {}) {
```

Comment:
Not really needed here at the moment, but maybe we can expand it in the future: https://github.com/digitalbazaar/bedrock-web-payment/pull/1/commits/ad2507115de81dcf96f940e9ce7172f59760240a

Reply:
Why do you believe it's not needed here at the moment? We want to consistently define our `urls` across all of our services we make in bedrock modules.

Reply:
ok did not know this was a pattern (despite the fact that I have written at least one of these before). TokenService also uses it: https://github.com/digitalbazaar/bedrock-web-authn-token/blob/master/TokenService.js

I just through it was not needed because we only have one URL. I'm assuming we will have more in the future?

Reply:
The subtlety here is the assumption that we only have one URL **right now**. That may not be true in the future. On top of that, this structure breaks away from the common pattern we want to have with web services. Personally, it is paramount to maintain the consistency of our libraries as much as possible. It reduces the cognitive load when picking up something new. If built properly, the library should feel familiar to a bedrock developer.